dot/parachain/network-bridge/collation_protocol_message.go

```diff
@@ -0,0 +1,56 @@
+package networkbridge
+
+import (
+	"fmt"
+
+	"github.com/ChainSafe/gossamer/dot/network"
+	collatorprotocolmessages "github.com/ChainSafe/gossamer/dot/parachain/collator-protocol/messages"
+	"github.com/ChainSafe/gossamer/pkg/scale"
+	"github.com/libp2p/go-libp2p/core/peer"
+)
+
+func decodeCollationMessage(in []byte) (network.NotificationsMessage, error) {
+	// TODO : Decode a wire message instead
```

Comment:
Are you planning to address in this PR?
If not, can you raise an issue for this?

Reply:
tracking this in a separate PR https://github.com/ChainSafe/gossamer/pull/4188
issue is #4108
added issue number in this TODO as well.